specification/cognitiveservices/HealthInsights/healthinsights.common/primitives.tsp

```diff
@@ -1,17 +1,21 @@
+import "@azure-tools/typespec-azure-core";
+
 using Azure.Core;
 using TypeSpec.Http;
+
 namespace AzureHealthInsights;
 
 #suppress "@azure-tools/typespec-azure-core/long-running-polling-operation-required" "This is a template"
 @doc("Long running RPC operation template")
-op LongRunningRpcOperation<
+@post
+op TLongRunningRpcOperation<
   TParams,
   TResponse
 > is Azure.Core.Foundations.Operation<
   TParams & RepeatabilityRequestHeaders,
-  (Foundations.AcceptedResponse<Foundations.LongRunningStatusLocation &
+  (Foundations.AcceptedResponse<Foundations.LongRunningStatusLocation<TResponse> &
```

Comment:
Not sure the template name `TResponse` is proper? According to `LongRunningStatusLocation` definition, the template indicates the real `TStatusResult` under the polling  reponses which belongs to the `result` column.

```
@doc("Metadata for long running operation status monitor locations")
model LongRunningStatusLocation<TStatusResult = never> {
  @pollingLocation
  @doc("The location for monitoring the operation state.")
  @TypeSpec.Http.header("Operation-Location")
  operationLocation: ResourceLocation<OperationStatus<TStatusResult>>;
}
```

https://github.com/Azure/typespec-azure/blob/main/packages/typespec-azure-core/lib/foundations.tsp#L101-L107